tests/test_verifylib.py

```diff
@@ -395,22 +395,31 @@ def setUp(self):
     self.materials = {
       "foo": {"sha256": self.sha256_foo},
       "foobar": {"sha256": self.sha256_foobar},
+      "sub/foobar": {"sha256": self.sha256_foobar},
       "sub/foo": {"sha256": self.sha256_foo},
-      "sub/foobar": {"sha256": self.sha256_foobar}
-
+      "lib/build/foobar": {"sha256": self.sha256_foobar},
+      "lib/foo": {"sha256": self.sha256_foo},
+      "lib/build/foo": {"sha256": self.sha256_foo},
+      "lib/foobar": {"sha256": self.sha256_foobar},
+      "build/lib": {"sha256": self.sha256_foobar}
```

Comment:
I think this isn't used in a test and is again technically a directory. We've used foo / bar / foobar etc for files.